packages/pg-protocol/src/serializer.ts

```diff
@@ -26,6 +26,7 @@ const startup = (opts: Record<string, string>): Buffer => {
   }
 
   writer.addCString('client_encoding').addCString('UTF8')
+  writer.addCString('extra_float_digits').addCString('1')
```

Comment:
I think this belongs in pg’s use of `startup` instead of in pg-protocol. (Side benefit: fewer/no other tests to update, since that’s at least part of why CI is failing right now.)

It also needs to be higher than 1 (take `0.42467013940651843` as a random example), with an accordingly stricter test case. I don’t have time right now to figure out exactly how much precision is enough, but setting it to 3 (https://www.postgresql.org/docs/11/runtime-config-client.html#GUC-EXTRA-FLOAT-DIGITS) is probably fine.